### PR TITLE
Use always HTTPS for request to our API. @brianr @coryvirok

### PR DESCRIPTION
**Pull request by jondeandres**

### CODE REVIEW

File: dist/rollbar.js [hunk 1]
```diff
@@ -254,10 +254,7 @@
 	  // when the rate limit is reached.
 	  _topLevelNotifier = _topLevelNotifier || this;
 	
-	  var protocol = window.location.protocol;
-	  if (protocol.indexOf('http') !== 0) {
-	    protocol = 'https:';
-	  }
+	  var protocol = 'https';
```
Comment (brianr) [inline dist/rollbar.js]: Looks like this variable is only used one place, the very next line. Any reason not to inline it in the next line?

**Comment by brianr** [comment]
A couple other issues here:

- The change should be made in the `src/` directory. `dist/rollbar.js` is a generated file.
- Build is failing (looks unrelated to this change, but we should get it passing before merge)

File: dist/rollbar.js (hunk 1 above)
Comment (jondeandres) [inline dist/rollbar.js]: I'll change it, ok.

**Comment by jondeandres** [comment]
@brianr oh yes, seems I pushed the change in dist/ instead of src/. Let me change it.

**Comment by jondeandres** [comment]
@brianr @coryvirok https://github.com/rollbar/rollbar.js/commit/8517ac44c5ab71d150789bbf76d89115bfb2761d

**Pull request merged**